lib/tilebag.rb

```diff
@@ -0,0 +1,52 @@
+module Scrabble
+  class TileBag
+    #player can see letters
+    attr_reader :letter_quantity
+
+
+    # Creating an instance of letter quantity to be used though class tilebag.
+    def initialize
+      @default_set = {A:9, B:2, C:2, D:4, E:12, F:2, G:3, H:2, I:9, J:1, K:1, L:4, M:2, N:6, O:8, P:2, Q:1, R:6, S:4, T:6, U:4, V:2, W:2, X:1, Y:2, Z:1}
```

Comment:
There's a problem with this structure.  It gives each letter an **equal** chance of being drawn despite the fact that there are 2 "M" tiles and 9 "A" tiles.  Uh oh!